net/sixlowpan/sixlowpan_input.c

```diff
@@ -616,11 +615,14 @@ static int sixlowpan_frame_process(FAR struct radio_driver_s *radio,
 
 static int sixlowpan_dispatch(FAR struct radio_driver_s *radio)
 {
+#ifdef CONFIG_NET_6LOWPAN_DUMPBUFFER
+  struct net_driver_s *dev = &radio->r_dev;
```

Comment:
Please put this below `int ret;` and hide `sixlowpan_dumpbuffer` under `#ifdef CONFIG_NET_6LOWPAN_DUMPBUFFER`

Reply:
Done